lib/bitcoin/IBitcoinConfig.ts

```diff
@@ -6,7 +6,6 @@ export interface IBitcoinConfig {
   bitcoinRpcUsername: string | undefined;
   bitcoinRpcPassword: string | undefined;
   bitcoinWalletImportString: string;
-  bitcoinFee: number;
```

Comment:
Need to also remove all other references of `bitcoinFee`, a quick repo search shows at least 2 more location requiring removal (`README.md` & `setup.sh`).